draft/makingopensourceprojectsthrive.md

```diff
@@ -0,0 +1,132 @@
+#title: How to make your open-source project thrive, with Andrey Petrov
+
+===
+
+##Abstract: Andrey Petrov ([@shazow](https://twitter.com/shazow)) spoke at last week's [Sourcegraph open-source meetup](http://www.meetup.com/Sourcegraph-Hacker-Meetup/) about lessons learned from his successes and "many failures" creating open-source projects. Andrey is author of the popular urllib3 library and several other popular open-source libraries on GitHub. During the day, he works at Briefmetrics and is also a YC alum, ex-Googler, and a cat person. He writes mostly in Python, JavaScript, and Go. 
+
+
+**If you have the time, you should watch the full video of Andrey's awesome and hilarious talk.**
+
+===
+
+[Watch the video of Andrey's talk](https://youtu.be/ScUIlbHnxGI)
+
+[Link to Andrey's slides](https://docs.google.com/presentation/d/13FtcUcurtKdq3UJ65-xlWo8C5S4v_xtCYa5C7-QhKgg/edit?usp=sharing)
+
+===
+
+####Why open source?
```

Comment:
Our audience already knows "why open source", so I think we can skip this section. Maybe the article should start with your final paragraph in this section, "Open-sourcing has many benefits"...